tests/_files/baseline/expected.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<files>
+ <file path="FileWithIssues.php">
+  <line number="10" hash="6bf70f0b8c461415955e2d3a97cfbe664f5b957b">
```

Comment:
@sebastianbergmann 

If I understand correctly, we combine issues and caching in a single file. I believe it makes sense from the perspective of maintaining the feature (less complex), but may be a bit unexpected from the point of view of a user.

### Reasons for changes in the baseline

For example, in projects in which I use static code analysis, I typically only regenerate the baseline (single step) instead of running a static code analysis first, then fixing or regenerating the baseline second (two steps).

Let's ignore structural changes!

With `phpstan/phpstan`, the regenerated baseline has **two** reasons to change:

- an issue has been added to the baseline
- an issue has been removed from the baseline

With `vimeo/psalm`, the regenerated baseline has **three** reasons to change:

- an issue has been added to the baseline
- an issue has been removed from the baseline
- the version number of `vimeo/psalm` has changed

With `phpunit/phpunit`, the regenerated baseline has **four** reasons to change:

- an issue has been added to the baseline
- an issue has been removed from the baseline
- the line number in which an issue has been reported has changed
- the hash of a line in which an issue has been reported has changed

Not sure if that bothers a user, but you need to look more closely at the diff of the baseline if you care about it, to find out if you have added or removed an issue to the baseline.

What do you think?